.github/workflows/linter.yml

```diff
@@ -512,12 +510,6 @@ jobs:
           command: clippy
           args: --manifest-path ${{ matrix.crates }} --target=${{ matrix.target }} --features concordium-std/build-schema --features concordium-std/wasm-test -- -D warnings
 
-      - name: Run cargo clippy with schema and tests requiring quickcheck
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --manifest-path ${{ matrix.withquickcheck }} --target=${{ matrix.target }} --features concordium-std/build-schema --features concordium-std/concordium-quickcheck -- -D warnings
```

Comment:
So the quickcheck feature should be enabled for this crate. I'm a bit confused now why this works where I needed to introduce it previously.

Reply:
Right, it works because you moved the quickcheck from a dev-dependency to a normal dependency in https://github.com/Concordium/concordium-rust-smart-contracts/pull/255